semver.js

```diff
@@ -793,6 +793,56 @@ function toComparators(range, loose) {
   });
 }
 
+exports.comparatorsIntersect = comparatorsIntersect;
+function comparatorsIntersect(compA, compB, loose, platform) {
```

Comment:
Is this better as a standalone function, or as a prototype method on Comparator?